src/components/Dashboard.tsx

```
@@ -0,0 +1,198 @@
+              borderRadius: '6px',
+              textDecoration: 'none',
+              fontSize: '14px',
+              fontWeight: '500',
+            }}
+          >
+            ⚙️ System Settings
+          </a>
+          <a
+            href="/api/health"
+            target="_blank"
+            style={{
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)